albatross/core/serialize.h

```diff
@@ -38,21 +39,52 @@ class SerializableRegressionModel : public RegressionModel<FeatureType> {
             model_fit_ == other.get_fit());
   }
 
-  // todo: enable if ModelFit is serializable.
-  template <class Archive> void save(Archive &archive) const {
+  /*
+   * Include save/load methods conditional on the ability to serialize
+   * ModelFit.
+   */
+  template <class Archive>
+  typename std::enable_if<valid_output_serializer<ModelFit, Archive>::value,
+                          void>::type
```

Comment:
this hurts me :( I feel like we should extend the character limit on our formatter.

Reply:
yeah kinda annoying.  I guess it's already a pretty difficult line to read, so doesn't make it that much worse :)